volatility3/framework/symbols/linux/extensions/elf.py

```diff
@@ -314,10 +392,26 @@ def dynamic_sections(self):
                 break
 
 
+class elf_linkmap(objects.StructType):
+    def get_name(self):
+        try:
+            buf = self._context.layers.read(self.vol.layer_name, self.l_name, 256)
+        except exceptions.PagedInvalidAddressException:
+            # Protection against memory smear
+            return None
```

Comment:
Possibly worth a log message (although an `is None` test will tell you, so depends on how common this kind of a thing is...).  5:)